tool/tsh/tsh.go

```diff
@@ -581,6 +600,14 @@ func Run(args []string, opts ...cliOption) error {
 		return trace.Wrap(err)
 	}
 
+	confOptions, err := loadConfig()
+	if err != nil {
+		fmt.Println("Failed to load tsh config: ", err)
+	}
```

Comment:
If I remove the config file completely, tsh logs this message:

> Failed to load tsh config:  open /Users/rav/.tsh/config/config.yaml: no such file or directory

I'm pretty sure we would like to avoid logging this, right? It seems like having no config file should let tsh continue as if `confOptions` was nil.

We could take inspiration from `kubeconfig.Load`:

https://github.com/gravitational/teleport/blob/84b64fe487f9a74c64f702b65e2dc7bca3eb42a5/lib/kube/kubeconfig/kubeconfig.go#L218-L220

A design decision I don't agree with, but at least the behavior would be consistent across different config files. ;)

Reply:
Ah yes - sorry didn't see this comment. I guess it's consistent but it's a poor user experience when replacing a binary on a patch change.